### DISCUSSION TITLE
Send Custom Data on Status 40x 50x

### DISCUSSION THREAD

**Discussion by blackmo18** [vercel/next.js#11254] [Help] [answered]
Hi how I can send a custom response and message on status 400,

```javascript

export const (req, res) {
 // external api call
 const response = await axios.post(...)
 //handle response
 res.status(200).json(response.data)
 // if has an error
 res.status(400).json(response.data)
}

```

on front-end, I don't receive the payload data if  not status 200

```javascript
try {
  let response = await.post(...)
  console.log(response.data)
} catch (error) {
  console.log(error) // i cant get the payload passed here if not status 200
}
```

**Comment by vassbence** [comment, marked as answer]
[Here is some help from the axios docs](https://github.com/axios/axios#handling-errors), but honestly you are better off not using axios for simple use-cases like the one you provided. On the front-end next.js [already polyfills fetch](https://nextjs.org/blog/next-9-1-7#new-built-in-polyfills-fetch-url-and-objectassign) and in the lambdas you can use [node-fetch](https://www.npmjs.com/package/node-fetch).
